dspace-api/src/main/java/org/dspace/curate/Curation.java

```diff
@@ -300,9 +300,17 @@ private void initGeneralLineOptionsAndCheckIfValid() {
         // scope
         if (this.commandLine.getOptionValue('s') != null) {
             this.scope = this.commandLine.getOptionValue('s');
-            if (this.scope != null && Curator.TxScope.valueOf(this.scope.toUpperCase()) == null) {
```

Comment:
This was a "never null" error.  Enum.valueOf throws an exception if the object has no such value.  It never returns 'null'.  The null test was not only pointless but invalid as well.